WordPress/Resources/de.lproj/Localizable.strings

```diff
@@ -4069,7 +4069,7 @@ translators: Block name. %s: The localized block name */
 /* The Jetpack view title for the success state */
 "Jetpack installed" = "Jetpack installiert";
 
-/* Title of the Jetpack powered banner. */
+/* No comment provided by engineer. */
```

Comment:
🤔  This seems strange… especially since [other Localizable.strings seems to have a comment for that one](https://github.com/wordpress-mobile/WordPress-iOS/pull/19145/files#diff-765655c28527a302225d25e8418167b90b7e53fbbdc0b012f7d64ec0cd06cc60R9863-R9864) 🤷

Reply:
Oh wait no, this one has "Jetpack powered" as the key, while the one I liked has `jetpack.branding.badge_banner.title` as key, so those are not the same, my bad. Still strange that the string comment was… removed compared to last time we pulled those?